tests/UiTPAS/Listeners/SendUiTPASMailsMailpitTest.php

```diff
@@ -108,7 +108,7 @@ public function testMailWasSentIntegrationCreatedWithContacts(
         string $subject
     ): void {
         if (empty($this->mailpitUri)) {
-            $this->markTestSkipped('MAILPIT_DSN is not set, skipping email tests.');
+            $this->markTestSkipped('MAILPIT_API_URL is not set, skipping email tests.');
```

Comment:
I guess another hard fail is needed here. Otherwise, you could get false positives when `MAILPIT_API_URL` is missing and not executed?